spockk-compiler-plugin/src/main/kotlin/io/github/pshevche/spockk/compilation/transformer/SpockkIrFactory.kt

```diff
@@ -22,7 +22,7 @@ import org.jetbrains.kotlin.ir.expressions.IrExpression
 import org.jetbrains.kotlin.ir.symbols.UnsafeDuringIrConstructionAPI
 
 @OptIn(UnsafeDuringIrConstructionAPI::class)
-internal class SpockkIrFactory(pluginContext: IrPluginContext) {
+internal class SpockkIrFactory(val pluginContext: IrPluginContext) {
```

Comment:
🧐 I am going to remove the `SpockkIrFactory` in a future PR, as it is pretty shallow. I think that `SpecRewriter` should just receive an instance of `ContextAwareIrFactory` and just implement some member functions that do the transformation. But let's leave it as-is for now. I will clean it up once this PR is merged.